### DISCUSSION TITLE
How to deal with expiring auth keys?

### DISCUSSION THREAD

**Discussion by muzicman0** [jamesog/tailscale-edgeos#12] [General]
How do you deal with expiring auth keys?  having to generate a new key every 90 days would be a nightmare!

**Comment by jamesog** [comment]
Personally I set my EdgeRouter to be non-expiring. As with anything there's a security trade-off for this convenience.

**Comment by muzicman0** [comment]
Yeah, turns out I didn't understand what the auth code was.  It is only used once to authorize the device, then is never needed again. So this is a non issue.  It does seem to use a lot of CPU on the ER-4 when I transfer a large file though (>45%).  And that was just transferring 1 file.  I will probably just set up a server internal to my network and advertise the subnets I want to use.
